picard/ui/ui_infodialog.py

```diff
@@ -1,132 +1,73 @@
 # -*- coding: utf-8 -*-
+
 # Automatically generated - don't edit.
 # Use `python setup.py build_ui` to update it.
 
-from PyQt4 import QtCore, QtGui
-
-try:
-    _fromUtf8 = QtCore.QString.fromUtf8
-except AttributeError:
-    _fromUtf8 = lambda s: s
-
-class ArtworkTable(QtGui.QTableWidget):
```

Comment:
Ah, the file was edited manually in 7451602f8400bb32206e423286475b56e4cffa27 (not good). So we'll have to move the custom code to somewhere else, without changing any behavior.